### PR TITLE
Update annotations of layers.py

### PR DESCRIPTION
**Pull request by ranqiu92**
@lcy-seso

### CODE REVIEW

File: python/paddle/trainer_config_helpers/layers.py [hunk 1]
```diff
@@ -6436,34 +6450,36 @@ def gated_unit_layer(input,
...
-        projection of the input.
-    :type inproj_bias_attr: ParameterAttribute | None
-    :param layer_attr: Attributes to tune the final output of the gated unit,
-        for example, error clipping threshold, dropout and so on. See
-        ExtraLayerAttribute for more details.
+    :param inproj_param_attr: The parameter attribute of the projection. See ParameterAttribute
+                              for details.
+    :type inproj_param_attr: ParameterAttribute
+    :param inproj_bias_attr: The bias attribute of the projection. If the parameter is set to
+                             False or something not type of ParameterAttribute, no bias is
```
Comment (lcy-seso) [inline python/paddle/trainer_config_helpers/layers.py]: 把 "something not type of ParameterAttribute," 改为  If this parameter is set False or an object whose type is not ParameterAttribute 吧~ 原来那句话有语法错误。

其它出现地方相同。

Comment (ranqiu92) [reply python/paddle/trainer_config_helpers/layers.py]: Done

File: python/paddle/trainer_config_helpers/layers.py [hunk 2]
```diff
@@ -6436,34 +6432,34 @@ def gated_unit_layer(input,
 
     :param input: The input of this layer.
     :type input: LayerOutput
-    :param size: output size of the gated unit.
+    :param size: The dimemsion of this layer's output.
```
Comment (lcy-seso) [inline python/paddle/trainer_config_helpers/layers.py]: a type: dimension.

Comment (ranqiu92) [reply python/paddle/trainer_config_helpers/layers.py]: Done

**Pull request merged**